File: Python/TREE/Height of Binary Tree.py
//https://www.geeksforgeeks.org/problems/height-of-binary-tree/1
class Solution:
    #Function to find the height of a binary tree.
    def height(self, root):
        # code here
        if root == None:
            return -1
        hl = 1+self.height(root.left)
        hr = 1+self.height(root.right)
        return max(hl,hr)

File: Cpp/TREE/Height of Binary Tree.cpp
//https://www.geeksforgeeks.org/problems/height-of-binary-tree/1
class Solution{
    public:
    //Function to find the height of a binary tree.
    int height(struct Node* root){
        // code here 
        if(root==NULL){
            return -1;
        }
        return 1+max(height(root->left),height(root->right));
    }
};
